app/Importer/LicenseImporter.php

```diff
@@ -27,11 +27,14 @@ protected function handle($row)
      * @since 4.0
      * @param array $row
      * @return License|mixed|null
+     * updated @author Jes Vinsmoke
+     * @since 6.1
+     *
      */
     public function createLicenseIfNotExists(array $row)
     {
         $editingLicense = false;
-        $license = License::where('name', $this->item['name'])
+        $license = License::where('serial', $this->item['serial'])
```

Comment:
I think to get the best version of this, we should do the combination of name x serial as the 'unique identifier'. As serial isn't required. You can do a `License::where()->where()` to get two AND'ed lookups.

Reply:
updated!